applications/serial_lte_modem/doc/NRFCLOUD_AT_commands.rst

```diff
@@ -183,200 +183,89 @@ Example
 
 ::
 
-  AT#XXNRFCLOUD=?
+  AT#XNRFCLOUD=?
 
-  #XNRFCLOUD: (0,1,2),<signify>
+  #XNRFCLOUD: (0,1,2),<send_location>
 
   OK
 
-nRF Cloud cellular location
-===========================
+nRF Cloud location
+==================
 
-The ``#XCELLPOS`` command runs the nRF Cloud cellular location service for location information.
+The ``#XNRFCLOUDPOS`` command sends a request to nRF Cloud to determine the device's location using information from the cellular network and/or Wi-Fi access points.
```

Comment:
In general, the use of a slash in running text is imprecise (it can mean both "or" and "and or", depending on context). Used in "and/or", the meaning is clear, but it then becomes the only place where we could use it. For consistency, we try to not use it.

Reply:
You could add ", or both" to the end of that particular sentence to keep the meaning without adding too many words.